### PR TITLE
Update pna.p4 to include packet length as optional parameter

### PR DESCRIPTION
**Pull request by usha1830**

### CODE REVIEW

**Comment by rst0git** [comment]
Should we also update `pna.p4` in https://github.com/p4lang/pna?

File: p4include/pna.p4 [hunk 1]
```diff
@@ -373,7 +373,7 @@ enum PNA_CounterType_t {
 @noWarn("unused")
 extern Counter<W, S> {
   Counter(bit<32> n_counters, PNA_CounterType_t type);
-  void count(in S index);
```
Comment (mbudiu-vmw) [inline p4include/pna.p4:376]: Has this been discussed in the smartNIC WG?

**Review by mbudiu-vmw — APPROVED** [review]

File: p4include/pna.p4 (hunk 1 above)
Comment (usha1830) [reply p4include/pna.p4:376]: No. I just created a PR for the same in pna repository.
https://github.com/p4lang/pna/pull/49

Comment (mbudiu-vmw) [reply p4include/pna.p4:376]: So let's wait for that pr to merge first.

**Comment by usha1830** [comment]
Closing this PR as the corresponding PR for pna https://github.com/p4lang/pna/pull/49 is also closed.

Based on discussion with @jafingerhut, we will create new methods for dpdk which accept packet length parameter and remove @optional qualifier from this parameter.

**Pull request closed without merging**